pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/onchistory/OncHistoryUploadService.java

```diff
@@ -152,6 +171,14 @@ protected Map<Integer, Integer> getParticipantIds(List<OncHistoryRecord> oncHist
                     this.realm, updateElastic);
             medIds.put(participantId, medId);
         }
+
+        if (!exitedParticipants.isEmpty()) {
+            log.error("Found {} exited participants: {} in Onc History Upload", exitedParticipants.size(), exitedParticipants);
+            throw new OncHistoryValidationException("One or more of the uploaded onc histories is associated with a withdrawn participant. "
+                    + " Please remove onc histories for these withdrawn participants " + exitedParticipants
+                    + " from the file and upload it again.");
```

Comment:
This will read better if `exitedParticipants` is added at the end of the error string, after a colon.